packages/manager/cypress/support/intercepts/nodebalancers.ts

```diff
@@ -23,3 +23,14 @@ export const mockGetNodeBalancers = (
     paginateResponse(nodeBalancers)
   );
 };
+
+/**
+ * Intercepts POST request to intercept nodeBalancer data.
+ *
+ * @param
+ *
+ * @returns Cypress chainable.
+ */
+export const interceptCreateNodeBalancers = (): Cypress.Chainable<null> => {
```

Comment:
```suggestion
export const interceptCreateNodeBalancer = (): Cypress.Chainable<null> => {
```
Extremely minor nit: still holding out hope that some day there will be batch create APIs where we can create multiple resources at once 😅